e2e-tests/spring-rest-mongo/src/test/java/org/evomaster/e2etests/spring/rest/mongo/foo/MongoEMFitnessTest.java

```diff
@@ -0,0 +1,42 @@
+package org.evomaster.e2etests.spring.rest.mongo.foo;
+
+import com.foo.spring.rest.mongo.MongoPersonsAppController;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.utils.RestTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class
+
+
+MongoEMFitnessTest extends RestTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+        RestTestBase.initClass(new MongoPersonsAppController());
+    }
+
+    @Test
+    public void testRunEM() throws Throwable {
+
+        runTestHandlingFlakyAndCompilation(
+                "MongoEMFitness",
+                "org.foo.spring.rest.mongo.MongoEMFitness",
+                10000,
+                (args) -> {
+                    args.add("--enableWeightBasedMutationRateSelectionForGene");
+                    args.add("false");
```

Comment:
will need to add the experimental parameters that will need to be put to `false` by default in `EMConfig`